src/autofeat/utils.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+
+
+def random_seed_generator(num_seeds: int, low: int = 0, high: int = 2**32 - 1):
```

Comment:
This returns a list, so it is not really a generator and the function name is misleading. Might be more efficient as an actual generator though, i.e., always returning individual numbers with `yield`. 
But there is also a `Random Generator` from numpy (see [here](https://numpy.org/doc/stable/reference/random/generator.html)) which seems to be doing something similar, so it's probably better to use this instead.

Reply:
This is a great suggestion, thank you! I replaced the custom function with Numpy's Random Generator as you suggested.